File: src/main/java/dtos/CreateReleaseResponseDTO.java
package dtos;

public class CreateReleaseResponseDTO {

	private String url;
	private String uploadUrl;
	private String assetUrl;

	public String getUrl() {
		return url;
	}

	public void setUrl(String url) {
		this.url = url;
	}

	public String getUploadUrl() {
		return uploadUrl;
	}

	public void setUploadUrl(String uploadUrl) {
		this.uploadUrl = uploadUrl;
	}

	public String getAssetUrl() {
		return assetUrl;
	}

	public void setAssetUrl(String assetUrl) {
		this.assetUrl = assetUrl;
	}

	@Override
	public String toString() {
		return "CreateReleaseResponseDTO [url=" + url + ", uploadUrl=" + uploadUrl + ", assetUrl=" + assetUrl + "]";
	}
	
}

File: dtos/CreateReleaseResponseDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GitHubReleasesCLI.dtos
{
    public class CreateReleaseResponseDTO
    {
        public string? Url { get; set; }

        public string? UploadUrl { get; set; }

        public string? AssetUrl { get; set; }
    }
}
